src/transaction/log_manager.c

```diff
@@ -1634,6 +1634,12 @@ log_initialize_passive_tran_server (THREAD_ENTRY * thread_p)
       return;
     }
 
+  passive_tran_server *const pts_ptr = get_passive_tran_server_ptr ();
+  assert (pts_ptr != nullptr);
+
+  log_lsa replication_start_redo_lsa
+  {
+  NULL_LSA};
```

Comment:
Alternative to fix indentation:
```suggestion
  log_lsa replication_start_redo_lsa = NULL_LSA;
```